README.md

```diff
@@ -42,7 +42,8 @@ Example Playbook
     - linux-system-roles.tlog
   vars:
     tlog_scope_sssd: some
-    tlog_users_sssd: recordeduser
+    tlog_users_sssd:
```

Comment:
ok - but you also need to change tests/tests_example.yml:
```yaml
  vars:
    tlog_scope_sssd: some
    tlog_users_sssd: recordeduser
```

Reply:
Thanks, updated